main/observing/condition_checker.py

```diff
@@ -212,7 +212,9 @@ def rain_check(self):
                 urllib3.exceptions.InvalidHeader, requests.exceptions.ConnectionError, requests.exceptions.Timeout,
                 requests.exceptions.HTTPError):
             return None
-        api_key = re.search(r'"SUN_V3_API_KEY":"(.+?)",', self.radar.text).group(1)
+
+        #api_key = re.search(r'"SUN_V3_API_KEY":"(.+?)",', self.radar.text).group(1)
```

Comment:
I think I figured out the problem.  The backslashes are actually in the radar.text as double backslashes.  So try a re.search with \\\\" instead of \\" and see if that works!

Reply:
This worked for me:
api_key = re.search(r'\\\\"SUN_V3_API_KEY(.+?)\\\\":\\\\"(.+?)\\\\",', radar.text)
if api_key:
   api_key.group(2)

you should also add an else statement here like I mention in the other comment

Reply:
you might even want to add more versatility by changing it to re.search(r'\\\\"(.+?)API_KEY(.+?)\\\\":\\\\"(.+?)\\\\",', radar.text) and .group(3)